### DISCUSSION TITLE
Expected WiX v4 release date

### DISCUSSION THREAD

**Discussion by OrBin** [wixtoolset/issues#7229] [Questions]
Hi,
Do you have an expected release date for WiX v4?
I see that a month has passed since `v4-rc.2` and all issues in the [v4.0 milestone](https://github.com/wixtoolset/issues/milestone/5) are done.

**Comment by robmen** [comment]
You can follow status on the [FireGiant blog](https://www.firegiant.com/blog/). [Latest blog entry](https://www.firegiant.com/blog/2023/2/9/wix-online-meeting-251-highlights/) talks about `-rc.3` and `-rc.4`.

**Comment by OrBin** [comment]
Thanks.
Should we expect `-rc.5` too?
Also, I can't see the expected date for WiX v4 GA, is it still unknown?

**Reply by robmen** [reply]
For details, you should watch the video. You'll get a better understanding exactly what we're thinking vs the summary.
